django_freeradius/tests/__init__.py

```diff
@@ -0,0 +1,164 @@
+class CreateObjectsMixin(object):
```

Comment:
it would be better to rename this `CreateRadiusObjectsMixin` or something similar